mission_yaml/SEA45_M48.yml

```diff
@@ -373,11 +448,11 @@ profile_variables:
 
   instrument_ctd:
     comment:    unpumped RBR legato CTD
-    calibration_date:     "2021-03-01"
+    calibration_date:     "2020-07-23"
     calibration_report:  " "
     factory_calibrated:  "yes"
     long_name:           RBR legato seaexplorer CTD
     make_model:          RBR legato
     platform:            platform
-    serial_number:       "206523"
-    type:                platform
+    serial_number:       "205047"
```

Comment:
205048

Reply:
Man, did I get any of the serial numbers/calibration dates right? >< Really gotta learn to read one of these days!

Reply:
It can be confising :)